app/views/membership.html

```diff
@@ -5,30 +5,32 @@
     <div class="middle-container">
       <div class="middle-header">
         <div class="container-fluid">
-          <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
```

Comment:
If you remove the breadcrumbs from the view, we should also take the breadcrumbs out of the controller.